doc/installation-guide/server.rst

```diff
@@ -1,9 +1,15 @@
-************
-Requirements
-************
+Setup database server
+=====================
 
-Server-side
-===========
+If you expect to access the data from several different workstations, you can
```

Comment:
I'd replace "workstations" by "machines", but this may be just me :)

Reply:
Came from QGEP :) Well, i prefer workstation :D
whatever, for now...:)

Reply:
Yep